frontend/src/metabase/lib/colors/palette.ts

```diff
@@ -59,6 +59,7 @@ const aliases: Record<string, (palette: ColorPalette) => string> = {
   pulse: palette => color("accent4", palette),
 
   "brand-light": palette => lighten(color("brand", palette), 0.532),
+  "brand-lighter": palette => lighten(color("brand", palette), 0.598), // #EEF6FC for brand
```

Comment:
I'm really not sure this is a sustainable approach for this. Simply lightening the brand color like this works great for a certain set of colors (like metabase default brand), but for anything much lighter, it simply turns white.

Reply:
Should we just do `lighten` inline where we need to? We shouldn't introduce new palette colors until we have more components using them

Reply:
we actually have a bunch of these already, though they're all ever-so-slightly different values

https://github.com/search?q=repo%3Ametabase%2Fmetabase%20lighten(%22brand%22&type=code

design uses this pretty frequently as "baby blue" in figma.